app/register/page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+
+const goodUser = {
+  email: 'test1@email.com',
+  password: 'test12345',
+  confirmPassword: 'test12345',
+};
+
+const badUser = {
+  email: 'wrongemail@email.com',
+  password: 'wrongpassword',
+  confirmPassword: 'wrongpassword1',
+};
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/register');
+});
+
+test.describe('Tests register page', () => {
+  test('should successfully register', async ({ page }) => {
+    await page.getByTestId('email').fill(goodUser.email);
+    await page.getByTestId('password').fill(goodUser.password);
+    await page.getByTestId('confirm-password').fill(goodUser.confirmPassword);
+    await page.getByTestId('continue-button').click();
+    await page.goto('/weeklyPicks');
+    await expect(page).toHaveURL('/weeklyPicks');
+  });
+  test('should not be able to register and register button should be disabled', async ({
```

Comment:
Right now you're only testing for an incorrect password, but what if the email is invalid and the password is correct?

Reply:
Do we need to care about scenarios where the password inputs match initially (so button is enabled), but then one of the value changes? Should we check that the button's state reverts to disabled?

Reply:
That's a good scenario to consider too, Corina. 

@shashilo what are your thoughts on creating two additional test cases in regards to what Corina and Ryan stated?

Reply:
We need to check for all possible unsuccessful options.

Reply:
@vmaineng I'm not seeing the additional tests, unless I'm missing somehting.